src/components/JournalPreview/index.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
I don't think we should have a snapshot test for JournalPreview instead we should have unit test to test `JournalPreview` component because `JournalPreview` is wrapping `JournalPage` component for which we already have snapshot tests.